ImportError: <module 'main' from '/base/data/home/apps/s~gaetwythoncountdown2/1.397359309069680344/main.pyc'> has no attribute app

_LoadHandler (/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py:302)



ImportError: <module 'main' from '/base/data/home/apps/s~gaetwythoncountdown2/1.397359309069680344/main.pyc'> has no attribute app
at _LoadHandler (/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py:302)
at Handle (/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py:240)

Traceback (most recent call last):
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 240, in Handle
    handler = _config_handle.add_wsgi_middleware(self._LoadHandler())
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 302, in _LoadHandler
    raise err
ImportError: <module 'main' from '/base/data/home/apps/s~gaetwythoncountdown2/1.397359309069680344/main.pyc'> has no attribute app
